util/tamer.py

```diff
@@ -175,9 +196,13 @@ def _find_id(soup: BeautifulSoup) -> str:
     return str(id)
 
 
-def _find_full_id(soup: BeautifulSoup) -> str:
+def _find_full_id(soup: BeautifulSoup, currMS: str = 'none') -> str:
     id_raw = soup.find('msDesc')
-    id = id_raw.get('xml:id')
+    try:
+        id = id_raw.get('xml:id')
+    except:
+        id = 'none'
+        print(currMS)
     return str(id)
```

Comment:
1: It happens with a few MSs and the print debug thing was my way of trying to find out on which. Of course it printed `None`... Problem is: It kills handler construction because it will throw an exception and stop when trying to `.get()` on a None Type Object.

2: Totally right and we should stick to a standard way of error handling

Reply:
I'm surprized they have XMLs without ID, but ok... everything's possible, here

Reply:
I tried to fix it but as far as I can see it only affects SÁM 333 (is). Not putting in that much effort for a single MS atm. (It's specified as unicode but uses non-unicode characters btw.)